examples/scatterplot/app.js

```diff
@@ -40,24 +40,28 @@ class Root extends Component {
   }
 
   _resize() {
-    this._onViewportChange({
+    this.setState({
       width: window.innerWidth,
       height: window.innerHeight
     });
   }
 
   _onViewportChange(viewport) {
+    delete viewport.width;
+    delete viewport.height;
     this.setState({
-      viewport: {...this.state.viewport, ...viewport}
+      viewport
     });
   }
 
   render() {
-    const {viewport, data} = this.state;
+    const {viewport, data, width, height} = this.state;
 
     return (
       <MapGL
         {...viewport}
+        width={width}
```

Comment:
Should be able to just use `viewport.width` and `viewport.height` if they are not removed